addon/components/vertical-collection/component.js

```diff
@@ -57,33 +93,48 @@ const VerticalCollection = Component.extend({
   containerSelector: null,
 
   // –––––––––––––– Performance Tuning
-  /*
-   * how much extra room to keep visible and invisible on
-   * either side of the viewport.
+  /**
+   * The amount of extra room in pixels to keep visible and invisible on
+   * either side of the viewport. This value needs to be greater than 0.
+   *
+   * @property bufferSize
+   * @type Number
+   * @default 1
    */
   bufferSize: 1,
 
   // –––––––––––––– Initial Scroll State
-  /*
+  /**
    * If set, upon initialization the scroll
    * position will be set such that the item
    * with the provided id is at the top left
    * on screen.
    *
    * If the item cannot be found, scrollTop
    * is set to 0.
+   * @property idForFirstItem
    */
   idForFirstItem: null,
 
-  /*
+  /**
    * If set, if scrollPosition is empty
    * at initialization, the component will
    * render starting at the bottom.
+   * @property renderFromLast
+   * @type Boolean
+   * @default false
    */
   renderFromLast: false,
 
-  /*
-   * If set, the collection will render all of the items passed in
+  /**
+   * If set to true, the collection will render all of the items passed into the component.
+   * This counteracts the performance benefits of using vertical collection, but allows for improved accessibility,
+   * since the high performance of vertical-collection is attributed to rendering only the visible elements of a list,
+   * which is not a viable solution in a screen-reader environment, for example.
```

Comment:
I might add a note about this being a viable way to render everything in fastboot / other SEO solutions as well as implementing a listener for `meta f` that toggles this to allow the page to be searchable.

Reply:
Also should note, we actually _do_ have viable occlusion for screen readers. It still needs to be tested out, but the `occluded-content` elements on either side of the list will contain text saying `and {n} items before/after`, and should be clickable. Clicking paginates the list, scrolling it to the next `n` items (for however many are rendered). Generic occlusion a11y OOTB 😁

Reply:
> as well as implementing a listener for meta f that toggles this to allow the page to be searchable.

Can you expand on this? I'm not sure what this means

Reply:
He means like if you do `cmd-f` to search the webpage, flipping this option would allow users to search the whole text content by rendering all of the content

Reply:
ahhh... got it.. don't use mac... :smile:

Reply:
> Also should note, we actually do have viable occlusion for screen readers. It still needs to be tested out, but the occluded-content elements on either side of the list will contain text saying and {n} items before/after, and should be clickable. Clicking paginates the list, scrolling it to the next n items (for however many are rendered). Generic occlusion a11y OOTB :grin:
 
I punted on adding this to the docs, but we should definitely include it in an accessibility section once tested :D